quirks/password-rules.json

```diff
@@ -362,6 +362,9 @@
     "lowes.com": {
         "password-rules": "minlength: 8; maxlength: 12; required: lower, upper; required: digit;"
     },
+    "lsacsso.b2clogin.com": {
+        "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit,[!#$%&*+?@^];"
```

Comment:
The commit description states:

> The rule added by this commit uses only the characters in the textual description, minus the "." (to remove the possibility of a ".@" sequence).

However, the password rule as written here allows for the `+` character, which is not in the textual description in the screenshot. Should we remove `+` from the rule?

Reply:
You're right! I think this must've been a mis-copy. Thanks for catching!

I've updated the character set to remove '+' and add '-' and '_' as special characters.